bundles/org.openhab.binding.miele/src/main/java/org/openhab/binding/miele/internal/handler/CoffeeMachineChannelSelector.java

```diff
@@ -92,13 +86,13 @@ public State getState(String s, DeviceMetaData dmd, MieleTranslationProvider tra
             return UnDefType.UNDEF;
         }
     },
-    SWITCH(null, "switch", OnOffType.class, false);
+    SWITCH("", "switch", OnOffType.class, false);
 
     private final Logger logger = LoggerFactory.getLogger(CoffeeMachineChannelSelector.class);
 
-    private static final Map<String, String> programs = Collections.<String, String> emptyMap();
+    private static final Map<String, String> programs = Map.of();
 
-    private static final Map<String, String> phases = Collections.<String, String> emptyMap();
+    private static final Map<String, String> phases = Map.of();
```

Comment:
It would also be nice when all the "ConstantNameCheck" findings are fixed by changing the constants to uppercase. :slightly_smiling_face: 

See also: https://ci.openhab.org/job/PR-openHAB-Addons/8865/Static_20Code_20Analysis_20Report/

Reply:
Thanks for the hint - done.